packages/dashboard/pages/[did]/index.tsx

```diff
@@ -18,7 +18,15 @@ import {
   getOperations,
 } from '../../services/sidetree-node-client-api';
 
-const Resolver: NextPage = () => {
+export async function getServerSideProps(context: any) {
+  const res = await fetch(`http://${context.req.headers.host}/api/1.0`);
```

Comment:
what is this doing?

Reply:
Getting ui config via server, getting props this way required absolute url, this was a way we could think of to get this since window or next/router isn't available over here for server

Reply:
Updated PR!